pynest/nest/lib/hl_api_spatial.py

```diff
@@ -848,6 +847,8 @@ def SelectNodesByMask(layer, anchor, mask_obj):
 def _draw_extent(ax, xctr, yctr, xext, yext):
     """Draw extent and set aspect ration, limits"""
 
+    import matplotlib.pyplot as plt
```

Comment:
Could you add a comment about why we import here and not at the top? It would not be obvious for me when looking through the code, and I think it would be easy to convert back if we are not reminded why it is here. Also for the imports below.